contribs/gmf/less/timeslider.less

```diff
@@ -0,0 +1,49 @@
+.gmf-time-slider {
+  @handle-width : 1rem;
+
+  // 28 px come from bs popover class .popover-content { padding: 9px 14px; }
+  min-width: @popover-max-width - 28px;
+
+  .ui-widget-content {
+    border: none;
+  }
+
+  .ui-slider-handle {
+    top: -0.7rem;
+    cursor: pointer;
+    margin-left: -@handle-width / 2;
+    box-shadow: 0.9px 0.9px 1px rgba(0, 0, 49, 0.2), 0px 0px 0.9px rgba(0, 0, 75, 0.2);
+    border: 1px solid rgba(0, 0, 30, 0.2);
+    height: 20px;
```

Comment:
Why so many `px` units and not standard (assigned variable) values in this file ?
Please check.